src/apify_client/clients/resource_clients/schedule.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, Dict, List, Optional
+
+from ..._errors import ApifyApiError
+from ..._utils import _catch_not_found_or_throw, _pluck_data_as_list
+from ..base.resource_client import ResourceClient
+
+
+class ScheduleClient(ResourceClient):
+    """Sub-client for manipulating schedules."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the ScheduleClient."""
+        super().__init__(*args, resource_path='schedules', **kwargs)
+
+    def get(self) -> Optional[Dict]:
+        """Return information about schedule.
+
+        https://docs.apify.com/api/v2#/reference/schedules/schedule-object/get-schedule
+
+        Returns:
+            The retrieved schedule
+        """
+        return self._get()
+
+    def update(self, new_fields: Dict) -> Optional[Dict]:
```

Comment:
It would be nice if this actually accepted the fields to update as named parameters instead of just a dict, as we discussed with Matěj Sochor. Unfortunately, it means duplicating the parameters in code as it stands right now, but since the API should not change too much, it hopefully won't be too much of a problem.